.npmrc

```diff
@@ -0,0 +1,2 @@
+always-auth = true
+email = "noel.sharpe@digital.homeoffice.gov.uk"
```

Comment:
Don't add this.

Reply:
email is required for npm to authenticate. I will put the value into a secret. What email address would  you suggest?